core/models2/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class Contributor(models.Model):
+    profile =  models.ForeignKey(to=Profile, related_name="profiles", null=True, blank=True)
+    name = models.CharField(max_length=70)
+    netid = models.CharField(max_length=70)
+
+class Image(models.Model):
+    name = models.CharField(max_length=70)
+    filepath = models.CharField(max_length=150)
+    caption = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+class Profile(models.Model):
+    acc = models.OneToOneField(Account)
+    name = models.CharField(max_length=70)
+    bio = models.TextField()#not sure how exactly to be represented
```

Comment:
Yup TextField is good here (https://stackoverflow.com/questions/7354588/django-charfield-vs-textfield) - it'll be somewhat lengthy and I don't think it's critical that we specify max size